src/audio/drc/drc_math_lib/asin/c_fixpt/typedef.h

```diff
@@ -0,0 +1,132 @@
+/*
+ * File: typedef.h
```

Comment:
This file probably is able to be common among all c-fixpt implementations?

Reply:
Yes

Reply:
Since each library function is a module in itself so typedef is added for every module. I agree typedef.c is common for all math library functions